File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int size;
int main(void)
{
    do{
        size = get_int("How big is the pyramid? ");
    }while(size < 1 || size > 8);

    int rowsLeft = size;

    for(int rows = 1; rows <= size; rows++){
        for(int space = rowsLeft - 1; space > 0; space--){
            printf(" ");
        }
        for(int left = 0; left < rows; left++){
            printf("#");
        }
        printf("  ");
        for(int left = 0; left < rows; left++){
            printf("#");
        }
        rowsLeft -= 1;
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
def main():
    size = 0

    while size < 1 or size > 8:
        size = int(input("How big is the pyramid? "))

    rowsLeft = size

    rows = 1

    for rows in range(size):

        for space in range(rowsLeft - 1):
            print(" ", end = "")

        for left in range(rows):
            print("#", end = "")

        print("  ", end = "")

        for right in range(rows):
            print("#", end = "")

        rowsLeft -= 1
        print("")

if __name__ == "__main__":
    main()